konrad/ozone.py

```diff
@@ -3,16 +3,18 @@
 
 import abc
 import logging
+import numpy as np
+from netCDF4 import Dataset
 from scipy.interpolate import interp1d
-
 from konrad.component import Component
-from konrad.utils import ozone_profile_rcemip, refined_pgrid
 
 __all__ = [
     'Ozone',
     'OzonePressure',
     'OzoneHeight',
     'OzoneNormedPressure',
+    'OzoneCariolle',
+    'OzoneSiRaChA'
```

Comment:
You could append a ``,`` to make it easier for version control 😉

Reply:
Done and rebased, sorry for my messy history (still)...